packages/firestore/rollup.config.lite.js

```diff
@@ -106,8 +106,8 @@ const allBuilds = [
         },
         include: ['dist/lite/*.js']
       }),
-      json(),
       sourcemaps(),
```

Comment:
Did it not work in the previous order?

Reply:
I _think_ I moved it so that it would work. But I'll double check because my memory is a bit fuzzy now.

Reply:
It actually makes no difference if I move sourcemaps() here. I've undone this change from the PR.